### ISSUE TITLE
[steamshovel] add artist which displays a 3D dreibein (Trac #954)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix3#948]
We should have an artist that displays a small 3D dreibein with three arrows pointing into the x,y,z directions of the IceCube coordinate system. The position of this guy should be configurable. The default should be the origin of the IceCube coordinate system. The length of the arrows should also be configurable. The lengths should correspond reflect actual physical units, ie. 10 m or 1 km, etc..

<details>
 <summary><em>Migrated from https://code.icecube.wisc.edu/ticket/954
, reported by hdembinski and owned by hdembinski</em></summary>
<p>

```json
{
    "status": "closed", 
    "changetime": "2015-07-22T03:14:01", 
    "description": "We should have an artist that displays a small 3D dreibein with three arrows pointing into the x,y,z directions of the IceCube coordinate system. The position of this guy should be configurable. The default should be the origin of the IceCube coordinate system. The length of the arrows should also be configurable. The lengths should correspond reflect actual physical units, ie. 10 m or 1 km, etc..", 
    "reporter": "hdembinski", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1437534841447157", 
    "component": "combo core", 
    "summary": "[steamshovel] add artist which displays a 3D dreibein", 
    "priority": "normal", 
    "keywords": "steamshovel artist", 
    "time": "2015-04-30T20:05:08", 
    "milestone": "", 
    "owner": "hdembinski", 
    "type": "enhancement"
}
```

</p>
</details>

**Comment by icecube-trac** [comment]
Trac update at 20150512T14:13:48: hdembinski commented:

> Actually, mzoll already implemented that! Since it is such a trivial artist, however, I vote for rewriting it in Python and make it more flexible.

**Comment by icecube-trac** [comment]
Trac update at 20150722T03:14:01: 
* hdembinski changed status from "new" to "closed"
* hdembinski changed resolution from "" to "fixed"
* hdembinski commented:

> In changeset 135438:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="135438"
> python version of CoordinateSystem replaces c++ version, closes #948
> ```
